fingerprints/fingerprints.yaml

```diff
@@ -0,0 +1,821 @@
+
+      This likely indicates that Jinja template engine is being used."
+    metadata:
+      license: Semgrep Registry License
+    paths:
+      include:
+        - requirements.txt
+        - requirements.in
+        - setup.py
+    pattern-regex: "[Jj]inja"
+    severity: INVENTORY
+  - id: cai-python-dependency-sqlalchemy
+    languages:
+      - generic
+    message: "Detected `sqlalchemy` as a dependency in a requirements.txt file.
+
+      This likely indicates that database is being used."
+    metadata:
+      license: Semgrep Registry License
+    paths:
+      include:
+        - requirements.txt
+        - requirements.in
+        - setup.py
+    pattern-regex: "[Ss][Qq][Ll][Aa]lchemy"
+    severity: INVENTORY
+  - id: cai-python-framework-django
+    languages:
+      - python
+    message:
+      Detected `django.http` being imported, indicating this app uses the Python
+      Django web framework
+    metadata:
+      license: Semgrep Registry License
+    pattern: "import django.http.$FOO
+
+      "
+    severity: INVENTORY
+  - id: cai-python-framework-flask
+    languages:
+      - python
+    message:
+      Detected `Flask()` being instantiated, indicating this app uses the Python
+      Flask web framework
+    metadata:
+      license: Semgrep Registry License
+    pattern: "$FLASK = Flask(...)
+
+      "
+    severity: INVENTORY
+  - id: cai-ocaml-technology-opam
+    languages:
+      - generic
+    message: Detected a OPAM file. This likely indicates OCaml and OPAM are being used.
+    metadata:
+      license: Semgrep Registry License
+    paths:
+      include:
+        - "*.opam"
+    pattern-regex: ^.*
+    severity: INVENTORY
+  - id: cai-ocaml-technology-dune
+    languages:
+      - generic
+    message: Detected a dune file. This likely indicates OCaml and dune are being used.
+    metadata:
+      license: Semgrep Registry License
+    paths:
+      include:
+        - dune
+        - dune-project
+    pattern-regex: ^.*
+    severity: INVENTORY
+  - id: cai-ruby-dependency-rails
+    languages:
+      - generic
+    message: "Detected `rails` as a dependency in a Gemfile file.
+
+      This likely indicates that Ruby web framework Rails is being used."
+    metadata:
+      license: Semgrep Registry License
+    paths:
+      include:
+        - Gemfile
+    pattern-regex: gem ['"]rails
+    severity: INVENTORY
+  - id: cai-ruby-framework-rails
+    languages:
+      - ruby
+    message:
+      Detected a rails route being created, indicating this app uses the Ruby
+      rails web framework
+    metadata:
+      license: Semgrep Registry License
+    pattern: "Rails.application.routes.draw do \n  ...\nend\n"
+    severity: INVENTORY
+  - id: cai-generic-technology-nginx
+    languages:
+      - generic
+    message: Detected a .conf file. This likely indicates that Nginx is in use.
+    metadata:
+      license: Semgrep Registry License
+    paths:
+      include:
+        - "*.conf"
```

Comment:
Hm actually aren't there a variety of files that end in `.conf`? Could we make this more precise?

Reply:
It came this way; I think it's probably OK for now